spark_client/cli/service_account_registry.py

```diff
@@ -183,11 +183,11 @@ def create_service_account_registry_parser(parser: ArgumentParser):
         if maybe_service_account is None:
             raise NoAccountFound()
 
-        # maybe_service_account.configurations.log(logging.info)
-        logging.info(maybe_service_account.id)
+        # maybe_service_account.configurations.log(logging.critical)
+        logging.critical(maybe_service_account.id)
```

Comment:
As before, and all the rest of the logging is worth reverting

Reply:
as explained in the previous comment, we might need to 
- either revert to the bare print statement or
- make default log level as INFO or DEBUG and use INFO for these statements.